samples/web/content/apprtc/appwindow.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
```

Comment:
Yes, planning to do this, if it is useful, when we create the build script that packages the app. It isn't clear to me yet how similar index.html and appwindow.html wll be after we add all the new UI we are currently discussing.